### PR TITLE
Tici hardware abstraction layer

### PR DESCRIPTION
**Pull request by pd0wm**

### CODE REVIEW

File: common/hardware.py [hunk 1]
```diff
@@ -50,5 +51,7 @@ def get_network_strength(self, network_type):
 
 if EON:
   HARDWARE = cast(HardwareBase, Android())
```
Comment (adeebshihadeh) [inline common/hardware.py:53]: elif, otherwise android is broken

File: common/hardware.py [hunk 2]
```diff
@@ -50,5 +51,7 @@ def get_network_strength(self, network_type):
...
+if TICI:
```
Comment (adeebshihadeh) [inline common/hardware.py:54]: elif, otherwise android is broke

Comment (pd0wm) [reply common/hardware.py:54]: oops

File: common/hardware_tici.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+from cereal import log
+
+
+NetworkType = log.ThermalData.NetworkType
+NetworkStrength = log.ThermalData.NetworkStrength
+
+
+class Tici(HardwareBase):
+  @staticmethod
+  def get_cmdline():
```
Comment (adeebshihadeh) [inline common/hardware_tici.py:12]: not specific to tici, maybe move to hardware base class?

**Pull request merged**